docs/uml_directive/uml.py

```diff
@@ -0,0 +1,34 @@
+import os
+
+from docutils import nodes
+from docutils.parsers.rst import Directive
+
+
+class UmlImagesDirective(Directive):
```

Comment:
@TimdeBeer1 would it be possible to use the updated directive in `commonground-api-common` here as well? Or are there some customizations needed here?